PressureSensorAdapter.java

```diff
@@ -0,0 +1,43 @@
+
+/*					Author : Francois Mukaba
+ * 					Course : Software Engineering
+ * 					Spring 2019
+ */
+import sensor.PressureSensor;
+import java.awt.Color;
+
+public class PressureSensorAdapter implements SensorAdapter {
+	private PressureSensor p;
+
+	public PressureSensorAdapter() {
+		p = new PressureSensor();

```

Comment:
and then this line would become
```suggestion
		this.p = p;
```